docs/src/modules/ROOT/pages/llms.txt

```diff
@@ -0,0 +1,137 @@
+
+## Tutorials
+
+- [Author your first Akka service](https://doc.akka.io/java/author-your-first-service.html): Step-by-step guide to creating your first complete Akka service from scratch.
+- [Shopping Cart](https://doc.akka.io/java/shopping-cart/index.html): Overview of the shopping cart tutorial series that demonstrates core Akka SDK concepts.
+- [Shopping Cart - Quickstart](https://doc.akka.io/java/shopping-cart/quickstart.html): Get a shopping cart application up and running with event sourced entities.
+- [Shopping Cart - Adding a view](https://doc.akka.io/java/shopping-cart/addview.html): Extending the shopping cart application with a read-optimized view.
+- [AI RAG Agent](https://doc.akka.io/java/ask-akka/index.html): Introduction to building a Retrieval Augmented Generation (RAG) agent with Akka.
+- [AI RAG Agent - Knowledge indexing with a workflow](https://doc.akka.io/java/ask-akka/indexer.html): Implementing the knowledge indexing process for a RAG agent using Akka workflows.
+- [AI RAG Agent - Modeling the agent session](https://doc.akka.io/java/ask-akka/session.html): Creating a stateful, conversational AI agent using event sourced entities.
+- [AI RAG Agent - Executing RAG queries](https://doc.akka.io/java/ask-akka/rag.html): Performing contextual information retrieval and generating responses with a large language model.
+
+## Examples
+
+- https://github.com/akka-samples/ask-akka-agent AI agent that creates personalized travel itineraries. Illustrates reliable interaction with a LLM using Langchain4J and a workflow. Entities are used for durable state of user preferences and generated trips.
+- https://github.com/akka-samples/medical-tagging-agent AI agent that leverages a LLM to process medical discharge summaries and assign tags, while also enabling human verification and comparative analysis. Interactions with OpenAI LLM using the OpenAI Java SDK and a workflow.
+- https://github.com/akka-samples/changelog-agent AI agent that creates release notes summaries, every time there is a release from set up GitHub repositories. Interactions with Anthropic Claude is using the Anthropic Java SDK. Using tools to retrieve detailed information from GitHub. Entities are used for storing release summaries. Timed action looks for new releases periodically and creates the summary using the LLM.
+- https://github.com/akka-samples/shopping-cart-quickstart Build a shopping cart
+- https://github.com/akka-samples/event-sourced-customer-registry A customer registry with query capabilities
+- https://github.com/akka-samples/transfer-workflow-compensation A funds transfer workflow between two wallets
+- https://github.com/akka-samples/choreography-saga-quickstart A user registration service implemented as a Choreography Saga
+- https://github.com/akka-samples/akka-chess Akka Chess is a sample application that illustrates building a complete, resilient, automatically scalable, event-sourced application using the Akka SDK.
+
+## Code Style Guidelines
```

Comment:
I wonder if we should mention the package convention. `module.domain, module.application, ...` ?